deployment/tc532411/history-limit-dc.yaml

```diff
@@ -30,4 +30,4 @@ spec:
         - "100"
   triggers:
   - type: ConfigChange
-  revisionHistoryLimit: 3
+  revisionHistoryLimit: 2
```

Comment:
Use this file to test the function about auto clean the old RC, if revisionHistoryLimit is 3, we need to deploy 5 times to clean the first RC. if revisionHistoryLimit is 2, we only deploy 4 times.